### PR TITLE
Update finished portfolio to fix errors for deployment

### PR DESCRIPTION
**Pull request by rylan-marianchuk**
I have completed my portfolio. It is a lot of code for one commit, this is because I already had developed most of the content last year. However the new files I modified this week are 'projects.html' and its corresponding styles at 'styles/projects.css' where I updated the list view for project titles. For the code review, you could ignore all other files.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
+<body>
+    <div class="top-nav">
+        <button class="trigger" style="margin-top: 0;"><img src="img/MenuIcon.png" style="max-height: 100%; max-width: 100%; width: 50%; height: 35%; pointer-events: none;"></button>
+        <div class="navbar">
+            <a href="#">R Y L A N  &nbsp; M A R I A N C H U K</a>
+        </div>
+    </div>
+
+    <ul class="side-menu">
+        <li class="menu-items"><a href="#"> H O M E </a></li>
```
Comment (imadkar) [inline portfolio/src/main/webapp/index.html:19]: I'd recommend creating a css style using the letter-spacing [property](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing). This way you can keep the url as "Home" which is more semantic and allows crawlers and screenreaders to work better!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+        <li class="menu-items"><a href="projects.html">P R O J E C T S</a></li>
+        <li class="menu-items"><a href="personal2.html">B I O</a></li>
+    </ul>
+    <div class="side-nav"></div>
```
Comment (imadkar) [inline portfolio/src/main/webapp/index.html:23]: Maybe add a comment here and the other pages on why this is an empty div?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,36 @@
...
+    <title>Home</title>
+    <link rel="stylesheet" type="text/css" href="styles/nav.css">
+    <link rel="stylesheet" type="text/css" href="styles/home.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="top-nav">
+        <button class="trigger" style="margin-top: 0;"><img src="img/MenuIcon.png" style="max-height: 100%; max-width: 100%; width: 50%; height: 35%; pointer-events: none;"></button>
+        <div class="navbar">
+            <a href="#">R Y L A N  &nbsp; M A R I A N C H U K</a>
```
Comment (imadkar) [inline portfolio/src/main/webapp/index.html:14]: I'd recommend creating a style using [property](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing) to create your spaced letters effect here and the other pages. This way when a crawler for a search engine goes through your site, it'll parse your name correctly so the page will show up in search results for your name!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Home</title>
+    <link rel="stylesheet" type="text/css" href="styles/nav.css">
+    <link rel="stylesheet" type="text/css" href="styles/home.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="top-nav">
+        <button class="trigger" style="margin-top: 0;"><img src="img/MenuIcon.png" style="max-height: 100%; max-width: 100%; width: 50%; height: 35%; pointer-events: none;"></button>
```
Comment (imadkar) [inline portfolio/src/main/webapp/index.html:12]: I'd recommend pulling the styles into a css class. This way if you make a change you only need to modify the css class and not each page.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,18 +1,36 @@
...
+    </div>
+
+    <ul class="side-menu">
+        <li class="menu-items"><a href="#"> H O M E </a></li>
+        <li class="menu-items"><a href="projects.html">P R O J E C T S</a></li>
+        <li class="menu-items"><a href="personal2.html">B I O</a></li>
+    </ul>
+    <div class="side-nav"></div>
+
+
```
Comment (imadkar) [inline portfolio/src/main/webapp/index.html:25]: nit: this is an extra line

File: portfolio/src/main/webapp/personal2.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<div class="container">
+    <!--Grid row-->
+    <div class="row">
+
+        <!--Grid column-->
+        <div class="col-lg-4 col-md-12 mb-4">
+
+            <img src="https://gnrbla.bn.files.1drv.com/y4mGJHhoylOPoqmnunCy3C4FSCuXpXCeNgV6gyl6Zt26XOQXkDlsxuutCHNs--GDH0_elnPesusjYCgPLsivGxSsbcD-r3F_GN_5c6D_P1jZv5T7d6HLu-VdMzPrtTkDR1YdAhCCzQAtmLhtDvfk0iPhGywOqZLRbZU1RGsITyVbro_r91AkseaHlESu241FAycu24a_5bQx5YPElpuabMOyA?width=2848&height=4288&cropmode=none" class="img-fluid mb-4 rellax img_personal" alt="" data-rellax-speed="2" style="margin-top: 5vh;">
+
+            <!--  <img src="img/profile/2017-06-02%2010.18.07%201.jpg" class="img-fluid mb-4 rellax img_personal" alt="" data-rellax-speed="1" style="margin-top: 8vh;"> -->
```
Comment (imadkar) [inline portfolio/src/main/webapp/personal2.html:39]: Did you mean to remove this?

File: portfolio/src/main/webapp/personal2.html [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+        </div>
+        <!--Grid column-->
```
Comment (imadkar) [inline portfolio/src/main/webapp/personal2.html:42]: Were you going to add another column or should this be removed?

File: portfolio/src/main/webapp/styles/projects.css [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+    text-decoration: none;
+    margin:0;
+    background-color: rgba(98, 98, 98, 0.55);
+}
+
+.project-type:hover {
+    background-color: rgba(135, 135, 135, 0.8);
+
+}
+
```
Comment (imadkar) [inline portfolio/src/main/webapp/styles/projects.css:47]: nit: Could you remove this extra line?

**Comment by rylan-marianchuk** [comment]
Thanks for the review and suggestions Imad. It would be a good idea to not have my name puncuated with spaces throughout to have my name come up in search. Going forward I will not give you large amounts of code to review like this. I have committed (as the tutorial rerquired) a single change in the java Servlet file.

**Comment by rylan-marianchuk** [comment]
I have implemented a minimal functioning comment system on my index.html page. I have sectioned off the work into 5 steps as the tutorial has asked. The final commit I added css styling and query for permanent data storage.

**Pull request closed without merging**